### DISCUSSION TITLE
Newbie Question - How to use simple template in C# with Date time to create filename

### DISCUSSION THREAD

**Discussion by txprphan** [Handlebars-Net/Handlebars.Net#590] [Q&A]
I told you it was simple and I told you I was a newbie. Trying to replace Razor Engine scripts with Handlebars. This is the code I've got so far..

using HandlebarsDotNet;

var template = "MyFile_{{DateTime.Now \"MMddyyyy\"}}.txt";
var compiled = Handlebars.Compile(template);
var fname = compiled(new object {});

... and fname never has the date.

It's gotta be something simple, but like I said, I'm new to this.

**Comment by rexm** [comment]
I suggest you go through the README. It covers all the major functionality.
